src/Compilers/Test/Utilities/CSharp/SemanticModelTestBase.cs

```diff
@@ -220,5 +219,41 @@ protected CompilationUtils.SemanticInfoSummary GetSemanticInfoForTest(string tes
         {
             return GetSemanticInfoForTest<ExpressionSyntax>(testSrc);
         }
+
+        protected string GetOperationTreeForTest<TSyntaxNode>(CSharpCompilation compilation)
+            where TSyntaxNode: SyntaxNode
+        {
+            var tree = compilation.SyntaxTrees[0];
+            var model = compilation.GetSemanticModel(tree);
+            SyntaxNode syntaxNode = GetSyntaxNodeOfTypeForBinding<TSyntaxNode>(GetSyntaxNodeList(tree));
+            if (syntaxNode == null)
+            {
+                return null;
+            }
+
+            var operation = model.GetOperationInternal(syntaxNode);
```

Comment:
Yes, seems we need to expose the options when there are conflict - needs design discussion. You can probably add comment to #17861